resources/web/style/link.pcss

```diff
@@ -2,12 +2,10 @@
   a {
     color: #00a9e5;
     font-weight: normal;
-    -webkit-text-decoration: none;
     text-decoration: none;
     outline: none;
     outline-offset: 0;
     &:hover, &:focus {
-      -webkit-text-decoration: underline !important;
```

Comment:
Autoprefixer doesn't handle these at all because they haven't been needed in a long time.